testsuite/interaction_potentials/unittest/testFENE.py

```diff
@@ -20,11 +20,11 @@
 
 
 import unittest
-import espressopp.unittest
+import espressopp
 from espressopp.interaction.FENE import *
 from espressopp import Real3D, infinity
 
-class Test0FENE(espressopp.unittest.TestCase) :
+class Test0FENE(espressopp.tools.TestCase) :
```

Comment:
Is this test executed? Does not seem like that. Also `espressopp.tools.TestCase` does not exist?

Reply:
Occurs on multiple locations.

Reply:
Yup, this is executed. There are now 82 tests from the 66 in the master branch. Some were not executed previously. See: https://github.com/espressopp/espressopp/pull/372/checks?check_run_id=2519167875#step:9:54

I relocated the unittest module to tools since it was not being imported inside the testsuite directory. https://github.com/jnvance/espressopp/blob/ba953bd922fd78e3cf1c1cbb58f94f71d3e7d9e2/src/tools/test.py

Reply:
Ah, I missed the file you moved. Perfect.

I do not like that `espressopp.tools.TestCase` is not adhering to `unittest` naming, but that's a different issue.